xenmgr/Vm/Templates.hs

```diff
@@ -57,7 +57,7 @@ import Paths_xenmgr
 
 data ConfigTemplate = ConfigTemplate JSValue
 
-ndvmDefaultMode = "pv"
+ndvmDefaultMode = "hvm"
```

Comment:
Is a hvm ndvm required for 64bit?  I'm not opposed, but why is this change being made?

Reply:
Better to just use hvm until we can get pvh support because set of PRs makes everything 64-bit, including the NDVM.  If we don't want a x64 NDVM (or UIVM) we should have that discussion.

Reply:
Actually as a follow up, I'm going to test how well pvh UIVM works on x64

Reply:
This change makes a lot of sense, especially for a 64bits guest.
However, I'd recommend to do it in a separate PR, for a couple of reasons:
- Even if it's a one-liner, it's a big change, I think it deserves its own PR, build and tests.
- I have noticed that NDVM is slower to start as an HVM, especially on first boot after install, to the point where the user has to wait ~30 seconds before creating a first VM, otherwise the network options may not be available.

Reply:
@crogers1 My question is really, does a 64bit PV NDVM work?

@jean-edouard yes, it is slower.  I didn't think was *that* slow, but looking at a local machine, yes it is.  NDVM boots first though, so while everything waits for it, networking is available when starting guest.  Have you seen something different?

Reply:
I'm really talking about creating a guest, not starting it.

Reply:
@jandryuk yes, in the testing we've done so far, a 64bit PV NDVM does work.
@jean-edouard I've noticed something similar but haven't tracked down the cause yet.  Since nothing was technically broken I've been focused on other issues.